### PR TITLE
Add static library tutorial

### PR DESCRIPTION
**Pull request by pablogs9**

### CODE REVIEW

File: _docs/tutorials/advanced/create_custom_static_library/index.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+---
+title: Creating custom static micro-ROS library 
+permalink: /docs/tutorials/advanced/create_custom_static_library/
+---
+
```
Comment (FranFin) [inline _docs/tutorials/advanced/create_custom_static_library/index.md:5]: Before starting, I would add a couple lines describing what's the aim of this tutorial. Something like: "This tutorial aims at providing step-by-step guidance for those users interested in compiling micro-ROS as a standalone library. This procedure allows generating the code necessary for adding micro-ROS modules/components into external developing frameworks, and constitutes the basis for our [micro_ros_espidf_component](https://github.com/micro-ROS/micro_ros_espidf_component), [micro_ros_zephyr_module](https://github.com/micro-ROS/micro_ros_zephyr_module), and [micro_ros_arduino](https://github.com/micro-ROS/micro_ros_arduino) repos."

**Review by FranFin — CHANGES_REQUESTED** [review]

File: _docs/tutorials/advanced/create_custom_static_library/index.md (hunk 1 above)
Comment (pablogs9) [reply _docs/tutorials/advanced/create_custom_static_library/index.md:5]: Done

File: _docs/tutorials/advanced/create_custom_static_library/index.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+This tutorial aims at providing step-by-step guidance for those users interested in compiling micro-ROS as a standalone library in order to integrate it in custom development tools.
+
+This tutorial starts in a previously created micro-ROS environment. Check the first steps of [**First micro-ROS application on an RTOS**](../first_application_rtos/) for instructions about how to create a micro-ROS environment for embedded platforms.
```
Comment (FranFin) [inline _docs/tutorials/advanced/create_custom_static_library/index.md:8]:
```suggestion
This tutorial starts in a previously created micro-ROS environment. Check the first steps of [**First micro-ROS application on an RTOS**](../first_application_rtos/) for instructions on how to create a micro-ROS environment for embedded platforms.
```

**Review by FranFin — CHANGES_REQUESTED** [review]

File: _docs/tutorials/advanced/create_custom_static_library/index.md (hunk 2 above)
Comment (pablogs9) [reply _docs/tutorials/advanced/create_custom_static_library/index.md]: Done

**Review by FranFin — APPROVED** [review]

**Pull request merged**